modules/install-influxdb/README.md

```diff
@@ -3,7 +3,7 @@
 This folder contains a script for installing InfluxDB and its dependencies. Use this script to create an
 InfluxDB [Amazon Machine Image (AMI)](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html) that can be
 deployed in [AWS](https://aws.amazon.com/) across an Auto Scaling Group using the [influxdb-cluster
-module](https://github.com/gruntwork-io/terraform-aws-influx/tree/master/modules/influxdb-cluster).
+module](/modules/influxdb-cluster/README.md).
```

Comment:
Same for all of these. This is an open source repo that goes in the Terraform Registry, so these must be absolute URLs